### PR TITLE
Use by-lazy on TAG of SeedDatabaseWorker

### PR DESCRIPTION
**Pull request by XinyueZ**
Reason is that this TAG would only be used, only when there's error while operating on DB.

### CODE REVIEW

**Comment by tiembo** [comment]
LGTM, can you resolve the conflict?

**Comment by XinyueZ** [comment]
@tiembo resolved conflicts

File: app/src/main/java/com/google/samples/apps/sunflower/workers/SeedDatabaseWorker.kt [hunk 1]
```diff
@@ -27,8 +27,8 @@ import com.google.samples.apps.sunflower.data.AppDatabase
 import com.google.samples.apps.sunflower.data.Plant
 import com.google.samples.apps.sunflower.utilities.PLANT_DATA_FILENAME
 
-class SeedDatabaseWorker(context: Context, workerParams: WorkerParameters) : Worker(context, workerParams) {
-    private val TAG = SeedDatabaseWorker::class.java.simpleName
+class SeedDatabaseWorker(val context: Context, workerParams: WorkerParameters) : Worker(context, workerParams) {
```
Comment (tiembo) [inline app/src/main/java/com/google/samples/apps/sunflower/workers/SeedDatabaseWorker.kt]: Remove the `val` from context

**Comment by XinyueZ** [comment]
@tiembo have corrected, sry, my mist as merging of master.

**Comment by tiembo** [comment]
Thanks!

File: app/src/main/java/com/google/samples/apps/sunflower/workers/SeedDatabaseWorker.kt [hunk 2]
```diff
@@ -28,7 +28,7 @@ import com.google.samples.apps.sunflower.data.Plant
 import com.google.samples.apps.sunflower.utilities.PLANT_DATA_FILENAME
 
 class SeedDatabaseWorker(context: Context, workerParams: WorkerParameters) : Worker(context, workerParams) {
-    private val TAG = SeedDatabaseWorker::class.java.simpleName
+    private val TAG by lazy { SeedDatabaseWorker::class.java.simpleName }
```
[the remark this answers is not in the document]
Comment (XinyueZ) [reply app/src/main/java/com/google/samples/apps/sunflower/workers/SeedDatabaseWorker.kt]: @neiljaywarner The TAG will be used only when the Log.x will be used, that means, it is no need to occupy memory of this or calculated this *if* the TAG has never been used.

**Pull request merged**